sdk/util/rotation.cc

```diff
@@ -112,4 +112,46 @@ Rotation::VectorType Rotation::operator*(const Rotation::VectorType& v) const {
   return ApplyToVector(v);
 }
 
+double Rotation::GetYawAngle() {
+  // Variables used for a better understanding of the method
+  double x = quat_[0];
+  double y = quat_[1];
+  double z = quat_[2];
+  double w = quat_[3];
+
+  double siny_cosp = 2 * (w * y + z * x);
+  double cosy_cosp = 1 - 2 * (x * x + y * y);
+
+  return std::atan2(siny_cosp, cosy_cosp);
+}
+
+double Rotation::GetPitchAngle() {
+  // Variables used for a better understanding of the method
+  double x = quat_[0];
+  double y = quat_[1];
+  double z = quat_[2];
+  double w = quat_[3];
+  double pitch_angle;
+
+  double sinp = 2 * (w * x - y * z);
+  if (std::abs(sinp) >= 1)
```

Comment:
Nit: You can convert this into a one-liner:

```C++
return std::abs(sinp) >= 1. ? std::copysign(M_PI / 2., sinp) : std::asin(sinp);
```

Reply:
Also note that I added `.` next to `1` and `2` to make them floating point numbers.